src/Authentication/Authentication/custom/Find-MgGraphPermission.ps1

```diff
@@ -18,18 +18,23 @@ Find permissions for authorization against Microsoft Graph
 function Find-MgGraphPermission {
     [cmdletbinding()]
     param (
-        [string] $SearchString
```

Comment:
The comment with the synopsis field should be immediately on top of the `Find-MgGraphPermission` function.